app/views/bookings/_form.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= simple_form_for [@animal, @booking] do |f| %>
+  <%= f.input :date %>
+  <%= f.input :start_time %>
+  <%= f.input :duration %>
```

Comment:
This will need to be update to reflect the new database schema which no longer has the fields :date, :start_time, and :duration. The new ones will just be :start_date and :end_date. Besides that, looks good.